app/views/locations/welcome.html.erb

```diff
@@ -0,0 +1,3 @@
+<% content_for(:title, _("Locations")) %>
+
+<%= react_component('WelcomeLoc') %>
```

Comment:
This is no longer needed since you dropped it in your other PR (same for organizations and the associated components)